hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/PureWrapperOutputStream.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.hadoop.ozone.client.io;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Pure wrapper of OutputStream.
+ */
+public class PureWrapperOutputStream extends OutputStream {
+
+  private final OutputStream out;
+
+  public PureWrapperOutputStream(OutputStream out) {
+    this.out = out;
+  }
+
+  public void write(int b) throws IOException {
+    out.write(b);
+  }
+
+  public void write(byte[] b) throws IOException {
+    out.write(b);
+  }
+
+  public void write(byte[] b, int off, int len) throws IOException {
+    out.write(b, off, len);
+  }
+
+  public void flush() throws IOException {
```

Comment:
plz provide annotation @Override to indicate which one is from base

Reply:
Thank you @sumitagrawl, added the `@Override` annotation.